### DISCUSSION TITLE
customtkinter.Entry()

### DISCUSSION THREAD

**Discussion by sagareddum** [TomSchimansky/CustomTkinter#1329] [Q&A]
how to take input from users using customtkinter Entry without using textvariable
entry5=ctk.CTkEntry(f1, placeholder_text="      Conform-Password    ", text_color='grey',width=300,height=40,textvariable=a)
i am unable to use placeholder_text and textvariable at a time

**Comment by zSynctic** [comment]
You could remove the textvariable and just use ```entry5.get()``` but if you want to get the input when the user presses the enter key do this. Instead you bind the enter key so that when the user presses it it runs a function which gets the input and sets it to a variable:
```
import customtkinter

class App(customtkinter.CTk):
       def __init__(self):
           super().__init__()
           self.entry5=customtkinter.CTkEntry({your frame name}, placeholder_text=" Conform-Password ", text_color='grey',width=300,height=40)
           self.entry5.bind("<Return>", self.variable)
       
       def variable(self):
           global variable1
           variable1 = self.entry5.get()
```

**Comment by avalon60** [comment]
> how to take input from users using customtkinter Entry without using textvariable entry5=ctk.CTkEntry(f1, placeholder_text=" Conform-Password ", text_color='grey',width=300,height=40,textvariable=a) i am unable to use placeholder_text and textvariable at a time

Just use `entry5.get()`
